test/Mono.Linker.Tests/Tests/DocumentationSignatureParserTests.cs

```diff
@@ -571,6 +571,11 @@ public void NoGenericParameterType (Generic<A> g)
 			{
 			}
 
+			[ExpectUnresolvedDocumentationSignature ("T:Mono.Linker.Tests.DocumentationSignatureParserTests.Invalid.TypeWithMethodGenericParameters``1")]
```

Comment:
Could you add test for nested generic and non-generic types inside generic types as well?

Reply:
Generic types don't hit this edge case because they already have mangled names that will not match an unmangled method name. I will add the same test for a non-generic nested in a generic type.

Note that there are already nested generic tests for the happy case: https://github.com/mono/linker/blob/master/test/Mono.Linker.Tests/Tests/DocumentationSignatureParserTests.cs#L414